### ISSUE TITLE
Looks Broken-ish in iOS 9

### ISSUE THREAD

**Issue by nickoneill** [stripe/PaymentKit#80]
I just noticed this in an app that I built using the iOS 8 SDK on a phone that has been updated to iOS 9 (I don't really expect it to behave differently if built using the iOS 9 SDK but just an FYI for the state I'm in):

Since the field uses the standard system font, the widths are off using the new San Francisco and there are some ellipsis truncations happening. In addition, it looks like the color for placeholder text is darker than before? This makes the CC field look like it has already been filled in, this one might just be a 9b2 bug.

@jflinter I'm guessing your big rework of this isn't going to land before iOS 9?

I don't mean to suggest that this needs fixing immediately, particularly this early in the beta process. Just opening to track details.

**Comment by nickoneill** [comment]
Actually, I take that back. The switch to proportional-by-default only occurs when building with the iOS 9 SDK so we will probably see two different results initially for either SDK.

**Comment by jflinter** [comment]
Hey, thanks for opening this. I'll try and have something ready by the time
iOS9 drops.

**Comment by jflinter** [comment]
Hey @nickoneill,
I've got some good news - I've completed a large refactor of PaymentKit that works well with iOS 9. As part of the rewrite, the code has moved to https://github.com/stripe/stripe-ios, which now has a class called `STPPaymentCardTextField` that has similar behavior to `PTKView`. I'd encourage you to migrate (it should take just a few minutes, and we've written a brief guide at https://github.com/stripe/paymentkit#migration). Of course, if that doesn't work for you, or you have any trouble migrating, reply here and I'll be happy to help.
Cheers!
